app/helpers/application_helper.rb

```diff
@@ -106,7 +106,11 @@ def tag_colors(tag)
     end
   end
 
-  def any_selfserve_auth?
```

Comment:
Renamed this method based on some feedback on #9294 that seemed to slip through the cracks.